packages/loader/container-loader/src/container.ts

```diff
@@ -1964,6 +1964,7 @@ export class Container
 				true, // batch
 				message.metadata,
 				message.compression,
+				message.referenceSequenceNumber,
```

Comment:
I'm not sure if we want to allow different ref seq number for different ops in a batch. I.e. should it be a property of op, or a batch?
It feels like (with compression & possibly chunking requirements) we are settling on enforcing at runtime level that all ops in a batch should have same ref seq numbers. If so, we might want to pass ref seq number as an additional argument to submitBatch() API.

Reply:
working on it